internal/pkg/errors/error_message.go

```diff
@@ -154,6 +154,9 @@ const (
 	KafkaClusterShrinkingErrorMsg                 = "your cluster is shrinking; Please wait for that operation to complete before updating again"
 	KafkaClusterDeletingSuggestions               = ChooseRightEnvironmentSuggestions + "\n" +
 		"Ensure the cluster is not associated with any active Connect clusters."
+	KafkaClusterInaccessibleErrorMsg    = `Kafka cluster "%s" not found or access forbidden`
+	KafkaClusterInaccessibleSuggestions = ChooseRightEnvironmentSuggestions + "\n" +
```

Comment:
Do we have a test where this line is printed?

Reply:
The error message yeah, but not the new suggestion.

Reply:
Added a new int test, and I updated the delete suggestions to also include this new line.